cli/tests/default/e2e-pro/snapshots/test_ci/test_sarif_output_with_dataflow_traces/results.sarif

```diff
@@ -0,0 +1,694 @@
+=== command
+SEMGREP_APP_TOKEN="fake_key" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep ci --no-suppress-errors --dataflow-traces --sarif
+=== end of command
+
+=== exit code
+1
+=== end of exit code
+
+=== stdout - plain
```

Comment:
Probably should just call this file `results.txt` given that this is pretty clearly not a valid SARIF file.